tests/integration/test_subscriptions_cli.py

```diff
@@ -36,20 +36,31 @@
 # CliRunner().invoke(cli.main, args=['subscriptions', 'list', limit]
 #
 # does not work.
+
+
+@pytest.fixture
+def invoke():
```

Comment:
Nice call placing this in an `invoke` function, like we do for the Data and Orders CLI tests 👍